munkipkg

```diff
@@ -172,6 +172,9 @@ def get_build_info(project_dir):
         for key in supported_keys:
             if key in plist_info:
                 info[key] = plist_info[key]
+
+    if info.has_key('name'):
+        info['name'] = info['name'].replace('.pkg', '')
```

Comment:
I'd be happier with using .splitext() in case crazy people have a name like `my.pkgmaker.pkg` where your code would do the wrong thing